src/fr/envoyer.md

```diff
@@ -351,3 +340,147 @@ Si la demande a été refusée, le corps de la réponse est `json`, consultez le
+      "scheduled_for":null,
+      "sender_id":null,
+      "service":"afa2be3b-1250-430f-a70f-28a1a9d49dfa",
+      "service_name":{
+         "name":"Test service"
+      },
+      "template":"659a214f-dfec-4882-9242-fea0bd502a09",
+      "template_version":4,
+      "updated_at":null
+   }
+}
+```
+
+Vous pouvez suivre la progression de votre envoi en message depuis l'interface de GC Notification.
+
+Si vous avez programmé votre envoi, vous pouvez l'annuler depuis l'interface web.
+
+### Codes d’erreur
+
+Si la demande a été refusée, le corps de la réponse est `json`, consultez le tableau ci-dessous pour plus de détails.
+
+|Code HTTP|message|Comment réparer|
+|:---|:---|:---|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "You should specify either rows or csv"`<br>`}]`|Passez les données au moyen de `rows` ou `csv`|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "name is a required property"`<br>`}]`|Spécifiez le paramètre `name`|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "scheduled_for 42 is not of type string, null"`<br>`}]`|Vérifiez que vous passez une [date au format ISO 8601](https://fr.wikipedia.org/wiki/ISO_8601)|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "scheduled_for datetime cannot be in the past"`<br>`}]`|Vérifiez que vous passez une date future|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "scheduled_for datetime can only be up to 96 hours in the future"`<br>`}]`|Vérifiez que votre date est au plus 4 jours dans le futur|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "scheduled_for datetime format is invalid. It must be a valid ISO8601 date time format, https://en.wikipedia.org/wiki/ISO_8601"`<br>`}]`|Vérifiez que vous passez une [date au format ISO 8601](https://fr.wikipedia.org/wiki/ISO_8601)|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Template not found"`<br>`}]`|Mettez à jour l'identifiant de gabarit|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Template has been deleted"`<br>`}]`|Créez un nouveau gabarit et renseignez son identifiant|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Service is not allowed to send emails"`<br>`}]`|Activez l'envoi de courriels dans les Paramètres|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Missing column headers: name"`<br>`}]`|Ajoutez l'en-tête manquant|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Duplicate column headers: name, NAME"`<br>`}]`|Retirez l'en-tête en doublon|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Too many rows. Maximum number of rows allowed is 50000"`<br>`}]`|Renseignez moins de 50 000 lignes|
+|`400`|`[{`<br>`{"error": "BadRequestError",`<br>`"message": "You cannot send to these recipients because you used a team and safelist API key."`<br>`}]`|Demandez à activer votre service dans les Paramètres ou utilisez une [clé d'API active](cles.md)|
```

Comment:
```suggestion
|`400`|`[{`<br>`{"error": "BadRequestError",`<br>`"message": "You cannot send to these recipients because you used a team and safelist API key."`<br>`}]`|Demandez d'activer votre service dans les Paramètres ou utilisez une [clé d'API active](cles.md)|
```